apps/tx-builder/src/theme/SafeThemeProvider.tsx

```diff
@@ -1,17 +1,24 @@
-import { useEffect, useMemo, useState, type FC } from 'react'
+import React, { useEffect, useMemo, useState, type FC } from 'react'
 import { type Theme } from '@mui/material'
 import { ThemeProvider } from '@material-ui/core'
 import createSafeTheme from './safeTheme'
 import { getSDKVersion } from '@safe-global/safe-apps-sdk'
 
+export enum EModes {
+  DARK = 'dark',
+  LIGHT = 'light',
+}
```

Comment:
nit: Not sure what the E stands for. Is there a more descriptive name?

Reply:
E is for `Enum`, it is a typescript convention. Usually `T` for types and `I` for interfaces, so when you use the type, you would already know what it is without the needing of taking a look in the definition.